examples/Cargo.toml

```diff
@@ -6,8 +6,9 @@ edition = "2021"
 
 [dev-dependencies]
 relm4 = { path = "../relm4" }
-tokio = { version = "1.25", features = ["rt", "macros", "time", "rt-multi-thread"] }
```

Comment:
When running the example with the command `cargo run --example message_stream`, it uses the dependencies defined in relm4/Cargo.toml, but this file is being ignored. Is it necessary to keep this file?

Reply:
I never thought about this but it seems to work. Thanks!